lib/dateTime.js

```diff
@@ -622,16 +622,12 @@ function processPattern(pattern, replacements) {
 
 }
 
-export function convertUTCToLocalDateTime(date) {
-	if (!getDocumentLocaleSettings().timezone.identifier) {
-		return date;
-	}
-	const utcDate = new Date(Date.UTC(date.year, date.month - 1, date.date, date.hours, date.minutes, date.seconds));
+function convertJsDateToLocalDateTime(utcDate) {
```

Comment:
This should be in alphabetical order with the other helper functions. Is the change to returning either false or date object in order to avoid an extra `new Date` below here https://github.com/BrightspaceUI/intl/pull/76/files#diff-bfa90af68e38b31e2c7389bc93445f3bb17696889e6665f50c90c27d6813903cR1124 ? It seems like we should avoid a function returning two possible types since if this is used in the future a user would have to know to look out for that.

Reply:
Alphabetical: ✔️ 

For consistent return, here are some possibilities:
1. the function returns `null` instead of `false` - this is the smallest change
2. the function also takes a default value, and returns it if it can't do the conversion - in this case, `formatDateTimeFromTimestamp` ends up building a default object from a javascript `Date` only to immediately convert it back to the same `Date` - which is a bit on the strange and verbose side, but okay I guess
3. the function returns an object `{ result, isSuccess }`, and callers should use the result only if `isSuccess` is true - this is like the `Try...` pattern in C#, but in javascript I believe it's more idiomatic to do (1)
4. the function throws if it can't convert - seems like overkill

Do you have a preference among those?

Reply:
I think 1 or 2 would make the most sense. I'm fine with 1.

Reply:
Okay, done, and function renamed.